app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorViewController.kt

```diff
@@ -116,6 +117,7 @@ class SyncIndicatorViewController @Inject constructor(
     }
   }
 
+
```

Comment:
Am I supposed to read this commit message as "Call data sync for frequent group **only when** manual sync is triggered"? 

Or "Call data sync for **frequent group only** when manual sync is triggered"?

Reply:
Renamed.